experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -414,6 +414,20 @@ describe('Node SDK', () => {
       assert.equal(actualContextManager, expectedContextManager);
       await sdk.shutdown();
     });
+
+    it('should register a propagators as defined in OTEL_PROPAGATORS if trace SDK is configured', async () => {
```

Comment:
nit: typo
```suggestion
    it('should register propagators as defined in OTEL_PROPAGATORS if trace SDK is configured', async () => {
```

Reply:
good catch, fixed in https://github.com/open-telemetry/opentelemetry-js/pull/5355/commits/58ac3f0d918b175b15fce057e72b2ca3fe3b8a89